op_map.cc

```diff
@@ -292,7 +294,8 @@ struct OpMapperBase : public vx::op_map::IOpMapper {
       if (input_index < 0) {
         continue;
       }
-      if (context->tensors[input_index].type == kTfLiteInt64) {
+      if (context->tensors[input_index].type == kTfLiteInt64 &&
+          registration->builtin_code != 130) {
```

Comment:
130 is broadcast_to? Please add a comment why we can bypass type check for broadcast_to

Reply:
finished